spec/models/wunderground_history_spec.rb

```diff
@@ -1,79 +1,133 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe WundergroundHistory, :vcr do
-  let(:wunderground) { Wunderground.new(ENV['WUNDERGROUND_KEY']) }
-  let(:wh) { WundergroundHistory.new }
+  let(:wunderground) { Wunderground.new(ENV["WUNDERGROUND_KEY"]) }
+  let(:time) { Time.zone.parse("January 1, 2000 at 12am") }
+  let(:response) { wunderground.history_for(time, "knyc") }
 
-  describe "premature?" do
+  describe "error handling" do
 
-    it "returns true when no data yet available from wunderground" do
-      wh.time = Time.zone.parse('January 1, 3000 00:00:00 -04:00')
-      wh.response = wunderground.history_for(wh.time, 'knyc')
-      expect(wh).to be_premature
-    end
+    describe "validate!" do
+      context "when history is empty" do
+        let(:out_of_range_time) { Time.zone.parse("January 1, 3000 at 12am") }
+        let(:empty_response) { wunderground.history_for(out_of_range_time, "knyc") }
+        let(:empty_history) do
+          build(:wunderground_history, {
+            time: out_of_range_time,
+            response: empty_response
+          })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.